eth/vm/forks/constantinople/storage.py

```diff
@@ -0,0 +1,73 @@
+from eth.constants import (
+    UINT256
+)
+from eth.vm.forks.constantinople import (
+    constants
+)
+
+from eth.utils.hexadecimal import (
+    encode_hex,
+)
+
+
+def sstore_eip1283(computation):
```

Comment:
I moved this out into `constantinople/storage.py` because I strongly feel cluttering our main code base with fork specific logic is wrong. We currently have a bit of a mix (with the majority of fork specific logic being nicely isolated in `vm/forks/*`) Kinda related to our discussion we had in https://github.com/ethereum/py-evm/issues/1146

Reply:
I'm completely onboard with this.  I've got a code reorganization building up in my head to move all of the EVM stuff under `eth.evm` the same way the beacon chain stuff is under `eth.beacon`, which would also meaning moving a lot of stuff thats in `eth.validation`, `eth.constants`, `eth.rlp` etc around a bit to match.